packages/helix_editor.rb

```diff
@@ -6,7 +6,7 @@ class Helix_editor < Package
   version '22.12'
   license 'MPL-2.0' # license of source
   compatibility 'all'
-  source_url 'https://github.com/helix-editor/helix/archive/22.12.tar.gz' #'https://t.ly/-vk7'
+  source_url 'https://t.ly/-vk7'
```

Comment:
Please don't use a shortened url.

You can either use 
```
source_url 'https://github.com/helix-editor/helix.git'
git_hashtag version
```
or
```
source_url 'https://github.com/helix-editor/helix/archive/22.12.tar.gz'
source_sha256 'edae8af46401b45c3e71c38b4fa99f931c4458127978ccd1b29aaae79331d972'
```

Reply:
Yes ok. I did it because of another comment.
I prefer the normal path as it is more user friendly (or reader friendly I should say).

Reply:
Shorter is better, but not at the expense of obfuscating the source...

Reply:
ok removed now